lms/djangoapps/certificates/views/webview.py

```diff
@@ -271,7 +279,7 @@ def _update_social_context(request, context, course, user, user_certificate, pla
     context['twitter_share_enabled'] = share_settings.get('CERTIFICATE_TWITTER', False)
     context['twitter_share_text'] = share_settings.get(
         'CERTIFICATE_TWITTER_TEXT',
-        _("I completed a course at {platform_name}. Take a look at my certificate.").format(
+        _("I completed a course on #{platform_name}. Take a look at my certificate.").format(
```

Comment:
```suggestion
        _("I completed a course on {platform_name}. Take a look at my certificate.").format(
```

Right?

Reply:
I recovered the original line as like edx upstream